src/Components/WebView/Samples/PhotinoPlatform/src/BlazorWindow.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Reflection;
```

Comment:
Unneeded?

Reply:
I hate auto-namespace import for this reason.